issue found on: types/errors.go:127:1: paramTypeCombine

an issue has been detected in source code: ./types/errors.go:127:1: paramTypeCombine: func(regexStr string, str string) (string, error) could be replaced with func(regexStr, str string) (string, error)

--- a/types/errors.go
+++ b/types/errors.go
@@ -124,7 +124,7 @@ func regexGetNthMatchOrLogError(regexStr string, nMatch uint, str string) string
 
 	return match
 }
-func regexGetFirstMatch(regexStr string, str string) (string, error) {
+func regexGetFirstMatch(regexStr, str string) (string, error) {
 	return regexGetNthMatch(regexStr, 1, str)
 }
 

